src/lio/bin/lio_mv.c

```diff
@@ -76,7 +76,7 @@ gop_op_status_t mv_fn(void *arg, int id)
 //  tweak = (strcmp(mv->dest_tuple.path, "/") == 0) ? 1 : 0;  //** Tweak things for the root path
     while ((ftype = lio_next_object(mv->src_tuple.lc, it, &src_fname[slot], &prefix_len)) > 0) {
         snprintf(dname, OS_PATH_MAX, "%s/%s", mv->dest_tuple.path, &(src_fname[slot][prefix_len+1]));
-        gop = lio_move_op(mv->src_tuple.lc, mv->src_tuple.creds, src_fname[slot], dname);
+        gop = gop_lio_move_object(mv->src_tuple.lc, mv->src_tuple.creds, src_fname[slot], dname);
```

Comment:
Again where are the style docs?  I used the same naming convention of 
the exising routines in the same file. If we want to change it there 
needs to be a a guide we all agree to.  Reading tea leaves doesn't work.